tests/acceptance/shared/util/util.go

```diff
@@ -0,0 +1,199 @@
+package util
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"golang.org/x/crypto/ssh"
+)
+
+// RunCommandHost executes commands on the host
+// return the output of it and an error if any
+func RunCommandHost(cmd ...string) (string, error) {
+	var output bytes.Buffer
+	for _, cmd := range cmd {
+		c := exec.Command("bash", "-c", cmd)
+
+		stdoutPipe, err := c.StdoutPipe()
+		if err != nil {
+			return "", err
+		}
+		stderrPipe, err := c.StderrPipe()
+		if err != nil {
+			return "", err
+		}
+
+		err = c.Start()
+		if err != nil {
+			return "", err
+		}
+
+		_, err = io.Copy(&output, stdoutPipe)
+		if err != nil {
+			return "", err
+		}
+
+		_, err = io.Copy(&output, stderrPipe)
+		if err != nil {
+			return "", err
+		}
+
+		err = c.Wait()
+		if err != nil {
+			return output.String(), fmt.Errorf("error executing command: %s: %w", cmd, err)
+		}
```

Comment:
This copying may not work if the command hangs, or takes time to complete. If you don't want to use CombinedOutput(), because the need to keep stderr and stdout seperate, then the following would work better. 

```suggestion
        var output, errOut bytes.Buffer
        c.Stdout = &output
        c.Stderr = &errOut
        err := c.Run()
		if err != nil {
			return "", err
		}

        if eerb.Len() > 0 {
            return output.String(), fmt.Errorf("error executing command: %s: %s", cmd, errOut.String())
        }
```

Reply:
This is still not resolved. If you wish to keep it the same way, please provide a response as to why.